PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutCreditFinancingAmount.java

```diff
@@ -25,10 +25,6 @@ private PayPalNativeCheckoutCreditFinancingAmount() {}
     static PayPalNativeCheckoutCreditFinancingAmount fromJson(JSONObject amount) {
         PayPalNativeCheckoutCreditFinancingAmount result = new PayPalNativeCheckoutCreditFinancingAmount();
 
-        if (amount == null) {
-            return result;
-        }
-
```

Comment:
Let's revert this too

Reply:
Yeah we do own `Json.optString` though and it performs this check internally. I'm not sure why this guard exists, but I'm down to keep.

Reply:
Done: [bb94c54](https://github.com/braintree/braintree_android/pull/1059/commits/bb94c54728c389230f80a07d57728e4265b1aebb)